pyface/data_view/abstract_value_type.py

```diff
@@ -182,6 +183,49 @@ def set_text(self, model, row, column, text):
         """
         raise DataViewSetError("Cannot set value.")
 
+    def has_color(self, model, row, column):
+        """ Whether or not the value has color data.
```

Comment:
I don't want to say that this is setting the background colour, because that's just what it is doing right now; there are different ways that color data might be exposed in the future (eg. we could imagine a view which instead of setting the background color instead uses the color as the decoration; we have this use-case in real internal applications).

The value type is just providing a color, it's up to the view what is done with it.